runtime/doc/eval.txt

```diff
@@ -4031,9 +4031,12 @@ jobsend({job}, {data})					{Nvim} *jobsend()*
 			:call jobsend(j, ["abc", "123\n456", ""])
 < 		will send "abc<NL>123<NUL>456<NL>".
 
-jobstart({argv}[, {opts}])			{Nvim} *jobstart()*
-		Spawns {argv}(list) as a job. If passed, {opts} must be a
-		dictionary with any of the following keys:
+jobstart({cmd}[, {opts}])				{Nvim} *jobstart()*
+		Spawns {cmd} as a job.  If {cmd} is a |List|, it will be run
+		directly.  If {cmd} is a |String|, it will be equivalent to >
```

Comment:
The actual help option is `|string|`, so this should either be changed to that, or `*string*` in `eval.txt` should have `*String*` added next to it for consistency with `*List*` and `*list*`.